R/sparsevctrs.R

```diff
@@ -0,0 +1,3 @@
+is_sparse_tibble <- function(x) {
```

Comment:
is this automatically a tibble? I.e. can sparsevctrs vectors only exist in tibbles or is this actually more `has_sparse_vectors()`?

Reply:
it is. good point. sparse vectors can exists by themselves or in lists, data.frames or tibbles. This function will soon live in {sparsevctrs} itself, properly with a better name

https://github.com/r-lib/sparsevctrs/issues/70

Reply:
more specifically, `recompose()` when used in {recipes} always takes a tibble as input